### ISSUE TITLE
Errors with Singular 4.1.2.p5

### ISSUE THREAD

**Issue by zickgraf** [homalg-project/homalg_project#316]
With Singular 4.1.2.p5 the following error occurs:
```
gap> LoadPackage("RingsForHomalg");;
gap> F3xy := HomalgRingOfIntegersInSingular(3) * "x,y";;
gap> x1 := HomalgRingElement( "x+2", F3xy );;
gap> y0 := HomalgRingElement( "y", F3xy );;
gap> LoadPackage("LocalizeRingForHomalg");;
gap> R00 := LocalizeAtZero( F3xy );;
gap> R10 := LocalizeAt( F3xy, [ x1, y0 ] );;
gap> RMora := LocalizePolynomialRingAtZeroWithMora( F3xy );;
Error, the external CAS Singular (running with PID 10499) returned the following error:
   ? wrong range[2,2] in matrix _(1 x 1)
   ? error occurred in or before STDIN line 1852: `kill r;
```
This makes the CI fail. Older Singular versions seem to run fine.
Source of the example: `LocalizeRingForHomalg/examples/EasyPoly.g`

Note: I have temporarily downgraded Singular to make the CI pass again.

**Comment by mohamed-barakat** [comment]
Do we have this Singular version somewhere on plesken? I cannot figure out which Singular command is demanding such a range.

**Comment by zickgraf** [comment]
No, I cannot install the Singular update on plesken without updating the system as a whole. I could reinstall the update on homalgjenkins if you like.

**Comment by zickgraf** [comment]
Singular was updated automatically in the CI, so tests are failing again. How should we proceed?

**Comment by mohamed-barakat** [comment]
I have to get access to this version of Singular to be able to debug. I'll check it later today.

**Comment by mohamed-barakat** [comment]
Can we close this issue?

**Comment by zickgraf** [comment]
Yes, this is fixed in Singular 4.1.3.
